### PR TITLE
Document NISP test cases in portal and update bank profile

### PR DESCRIPTION
**Pull request by atamanroman**
# Summary

_(What is this feature about)_

# Checklist for Definition of Done

Depending on the task some of the following jobs are not relevant. Please check the items or cross them out (`~~`text`~~`).

* [x] Code is technically reviewed by myself
* [x] Code is manually tested
* [x] Pipeline is green
* [x] Documentation is updated
* [x] Branch is rebased and therefore ready to merge
* [x] Git history is clean

### CODE REVIEW

File: ui/src/app/dev-portal/developer-portal-page/developer-portal-page.component.html [hunk 1]
```diff
@@ -86,57 +112,93 @@ <h2>Getting started</h2>
...
-      </p>
-      <p class="description description__intro">Fictional PSU Data</p>
+      <ul>
+        <li>
+          To generate a QWAC Certificate follow this link
+          <a routerLink="/certificate-service">Certificate Service UI</a> and
+          embed the .pem and .key files in your requests.
+        </li>
+        <li>
+          The most popular Use Cases are described in the following section.
```
Comment (atamanroman) [inline ui/src/app/dev-portal/developer-portal-page/developer-portal-page.component.html]: Use Cases -> casing

File: ui/src/app/dev-portal/developer-portal-page/developer-portal-page.component.html [hunk 2]
```diff
@@ -86,57 +112,93 @@ <h2>Getting started</h2>
...
+          The following code snippet is an example cURL command which initiates
+          a single payment for PSU "PSU-Successful":
+        </p>
+        <pre><code>
 curl -v "https://sandbox-api.dev.adorsys.de/v1/payments/sepa-credit-transfers" \
   -H "accept: application/json" \
   -H "X-Request-ID: 99391c7e-ad88-49ec-a2ad-99ddcb1f7721" \
   -H "PSU-IP-Address: 192.168.8.78" \
   -H "Content-Type: application/json" \
+  -H "tpp-redirect-uri: http://kicker.de" \
```
Comment (atamanroman) [inline ui/src/app/dev-portal/developer-portal-page/developer-portal-page.component.html]: Let's use adorsys.de or so please.

File: ui/src/app/dev-portal/developer-portal-page/developer-portal-page.component.html [hunk 3]
```diff
@@ -86,57 +112,93 @@ <h2>Getting started</h2>
...
+        </li>
+      </ul>
+    </section>
+
+    <section>
+      <h2>Use Cases with PSU Data</h2>
       <p>
-        To execute services of XS2A, you have to use the following PSU
-        credentials
+        To execute services of XS2A using the EMBEDDED SCA approach, you have to
```
Comment (atamanroman) [inline ui/src/app/dev-portal/developer-portal-page/developer-portal-page.component.html:130]: Why do we mention EMBEDDED here?

File: ui/src/app/dev-portal/developer-portal-page/developer-portal-page.component.html [hunk 4]
```diff
@@ -86,57 +112,93 @@ <h2>Getting started</h2>
...
+      <h3>Simulation of SCA</h3>
+      <p>
+        SCA is mandated for the initiation and cancellation of payments as well
+        as for the creation of consents. To simulate the different SCA
+        responses, PSU-IDs and corresponding Ibans are defined. A successful SCA
+        can therefore only be achieved when performing the requests with the
+        Iban "DE11760365688833114935" and the corresponding PSU-ID
+        "PSU-Successful". The same goes for all other combinations of PSUs.
+        After sending your payment or consent request, you receive a scaRedirect
+        Url. Insert this url in your browser and attach your PSU-ID as
```
Comment (atamanroman) [inline ui/src/app/dev-portal/developer-portal-page/developer-portal-page.component.html]: Url url casing

File: ui/src/app/dev-portal/developer-portal-page/developer-portal-page.component.html (hunk 2 above)
Comment (isabellaio) [reply ui/src/app/dev-portal/developer-portal-page/developer-portal-page.component.html]: done

File: ui/src/app/dev-portal/developer-portal-page/developer-portal-page.component.html (hunk 4 above)
Comment (isabellaio) [reply ui/src/app/dev-portal/developer-portal-page/developer-portal-page.component.html]: done

File: ui/src/app/dev-portal/developer-portal-page/developer-portal-page.component.html (hunk 1 above)
Comment (isabellaio) [reply ui/src/app/dev-portal/developer-portal-page/developer-portal-page.component.html]: done

File: ui/src/app/dev-portal/developer-portal-page/developer-portal-page.component.html (hunk 3 above)
Comment (isabellaio) [reply ui/src/app/dev-portal/developer-portal-page/developer-portal-page.component.html:130]: resolved

**Pull request merged**